lib/pbench/test/unit/server/test_schema.py

```diff
@@ -354,23 +354,37 @@ def test_invalid_keyword(self, test):
         "type,keys,value,expected,delim",
         (
             (None, None, ["yes", "no"], ["yes", "no"], None),
-            (ParamType.KEYWORD, ["Yes", "No"], ["YeS", "nO"], ["yes", "no"], "|"),
+            (None, None, "one", ["one"], ";"),
             (
                 ParamType.ACCESS,
                 None,
                 ["Public", "PRIVATE"],
                 ["public", "private"],
                 None,
             ),
-            (ParamType.STRING, None, "yes,no", ["yes", "no"], ","),
-            (None, None, "one", ["one"], ";"),
+            (
+                ParamType.INT,
+                None,
+                ["1,2,3"],
+                [1, 2, 3],
+                ",",
+            ),
+            (
+                ParamType.INT,
+                None,
+                ["1", "2", "3"],
+                [1, 2, 3],
+                None,
+            ),
```

Comment:
Not going to try one where the incoming is already `int`??

Reply:
Given that Webb and I both approved despite this loose end ... I guess I'll resolve and move on.